traveltimepy/dto/responses/time_map_kml.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+from fastkml import Placemark, kml
+from pydantic import BaseModel
+
+# The stable version of fastkml is from 2021 and does not have types defined, nor
+# a `py.typed` file. There are newer pre-release versions as new as 2024, but they
+# seem to be undoccumented.
+# TODO: Maybe port this into newer versions of fastkml for proper type checking support
+
```

Comment:
Regarding this, the current version is `0.12`. They have 1.0 alpha versions available, seems like they have been slowly developing 1.0 for ~3 years now. It seems pretty vastly different and not documented, so for now I'll keep using the stable version with type checking disabled.